test/test_pickle_storage.py

```diff
@@ -40,6 +40,17 @@ def test_saved_load(self):
 
         assert_frame_equal(loaded_data, data)
 
+    def test_open(self):
```

Comment:
in the pickle storage there seems to be a function called `open` which opens a pickle file through pandas. Not sure what this is for since it is never used, but here is a test to make sure it works.

Reply:
I think it was used by BackTester to load backtest data